src/views/Dashboard/Customer/Home/RequestTest/MenuCard.vue

```diff
@@ -25,12 +27,14 @@
 
 import { mapState } from "vuex"
 import { getStates } from "@/common/lib/api"
+import debioLogo from "@/assets/debio-logo.png"
```

Comment:
why do we need this code? I agree with what @agustinustheo  said, we can directly use it without need the `import` in javascript as data